pallets/permissions/src/benchmarking.rs

```diff
@@ -0,0 +1,45 @@
+use crate::*;
+
+use polymesh_primitives::{DispatchableName, PalletName};
+
+use frame_benchmarking::benchmarks;
+use sp_std::{iter, prelude::*};
+
+const MAX_PALLET_NAME: u32 = 512;
+const MAX_DISPATCHABLE_NAME: u32 = 1024;
+
+fn make_name(m: u32) -> Vec<u8> {
+    iter::repeat(b'x').take(m as usize).collect::<Vec<_>>()
+}
+
+benchmarks! {
+    _ {}
+
+    set_call_metadata {
+        let p in 1..MAX_PALLET_NAME;
+        let d in 1..MAX_DISPATCHABLE_NAME;
+
+        let pallet_name: PalletName = make_name(p).into();
+        let pallet_name_exp = pallet_name.clone();
+        let dispatchable_name: DispatchableName = make_name(p).into();
+        let dispatchable_name_exp = dispatchable_name.clone();
+
+    }: {
+        StoreCallMetadata::<T>::set_call_metadata(pallet_name, dispatchable_name);
+    }
+    verify {
+        assert_eq!( CurrentPalletName::get(), pallet_name_exp);
```

Comment:
Can you please change all `assert`s to `ensure`s to make error output informative?